python/plugins/processing/algs/qgis/postgis_utils.py

```diff
@@ -153,7 +153,7 @@ def __init__(self, host=None, port=None, dbname=None, user=None,
         try:
             self.con = psycopg2.connect(self.con_info())
         except psycopg2.OperationalError as e:
-            raise DbError(unicode(e))
+            raise DbError(str(e))
```

Comment:
So I add a __unicode__ method on the DbError class, and encode it in utf8 in the __str__ method. It should work in all cases.